contracts/Swap/interfaces/CustomErrors.sol

```diff
@@ -42,5 +42,8 @@ error IdenticalAddresses();
 ///@notice Error thrown when the trade path[] parameter consists of only 1 token (i.e. path.length<2)
 error InvalidPath();
 
-//@notice Error thrown when invalid vTOken address is passed to swap router.
+///@notice Error thrown when invalid vTOken address is passed to swap router.
 error VTokenNotListed(address vToken);
+
+///@notice Error thrown when addresses path length is less than 2.
+error SwapInvalidAddressesPath(address[] path);
```

Comment:
Is it different from `InvalidPath` defined above? Maybe I miss something but the notices indicate these should be equivalent

Reply:
Fixed, d066bf93502d88423e3aafe3106bda584cef3103